tests/src/FunctionalJavascript/MessageFormTest.php

```diff
@@ -190,6 +190,7 @@ public function testCorporateForm(): void {
     $this->assertTrue(strpos($captured_emails[1]['body'], 'Test subject') !== FALSE);
```

Comment:
I think these are redundant with the new assertTestEmailBodies as $captured_emails[1]['body'] are already tested there